test/unit/sync/fixtures.py

```diff
@@ -11,14 +11,16 @@
 import pytest
 
 import apiver_deps
-from apiver_deps import AbstractFolder, File, B2File, B2FileVersion, FileVersionInfo
+from apiver_deps import AbstractFolder, File, B2File, B2FileVersion
 from apiver_deps import CompareVersionMode, NewerFileSyncMode, KeepOrDeleteMode
 from apiver_deps import DEFAULT_SCAN_MANAGER, Synchronizer
 
 if apiver_deps.V <= 1:
     from apiver_deps import FileVersion as VFileVersion
+    from apiver_deps import FileVersionInfo as VFileVersionInfo
 else:
     from apiver_deps import LocalFileVersion as VFileVersion
+    from apiver_deps import FileVersion as VFileVersionInfo
```

Comment:
I wanted to make to make sure that developers reading this are aware that this is not necessarily the `FileVersion` class defined in the SDK. It may or may not be it, depending on apiver. And because of how our tests are run, with `path` being manipulated before importing the test modules, IDEs don't point to proper classes via navigation features. So if you see a `FileVersion` used in a test case, you could be tempted to look at `b2sdk.file_version.FileVersion`. With my notation, it's clear that the class pointed to by this name depends on the apiver of the test.